tableaudocumentapi/xfile.py

```diff
@@ -70,6 +70,16 @@ def get_xml_from_archive(filename):
     return xml_tree
 
 
+def create_column(caption, datatype, role, field_type, name):
```

Comment:
This looks fine, but xfile is an odd place, I say toss it back in `fields.py` -- did @RussTheAerialist recommend it go here?